.github/workflows/docker.yaml

```diff
@@ -34,6 +34,7 @@ jobs:
         run: |
           python3 <<EOF
```

Comment:
nit: We could probable use `shell: python` instead of `<<EOF`.

Reply:
Oh, nice! I didn't know we could do that! Done!